core/src/main/java/org/jboss/pnc/causeway/ctl/ImportControllerImpl.java

```diff
@@ -172,10 +174,10 @@ public void untagBuild(TaggedBuild build, Request callback) {
     private BuildResult importBuild(Build build, String tagPrefix, String username, boolean reimport)
             throws CausewayException {
         if (build.getBuiltArtifacts().isEmpty()) {
-            throw new CausewayFailure("Build doesn't contain any artifacts");
+            throw new CausewayFailure(ErrorMessages.buildHasNoArtifacts());
         }
         if (!brewClient.tagsExists(tagPrefix)) {
-            throw new CausewayFailure(messageMissingTag(tagPrefix, config.getKojiURL()));
+            throw new CausewayFailure(ErrorMessages.tagsAreMissingInKoji(tagPrefix, config.getKojiURL()));
```

Comment:
How does this make the log message to become a 'WARN'?

Reply:
You are right, this doesn't :)
There was wrong log level at the place where this (Causeway**Failure**) exception is catched:
https://github.com/project-ncl/causeway/pull/161/files#diff-a87944d94ed4b24aff0eaf4e2b3fb05099d2c8a731bd6c63ed6cd49f55951978R149

Note that in the second class, the catching was done right already: https://github.com/project-ncl/causeway/pull/161/files/778e760c4777b791140a4e02e9ed2801c38bacd8#diff-a87944d94ed4b24aff0eaf4e2b3fb05099d2c8a731bd6c63ed6cd49f55951978R149